MSAL/src/MSALPublicClientApplication.m

```diff
@@ -268,7 +268,7 @@ - (id)initWithClientId:(NSString *)clientId
     MSALPublicClientApplicationConfig *config = [[MSALPublicClientApplicationConfig alloc] initWithClientId:clientId redirectUri:redirectUri authority:authority];
     
 #if TARGET_OS_IPHONE
-    config.cacheConfig.keychainSharingGroup = keychainGroup ?: [[NSBundle mainBundle] bundleIdentifier];
+    config.cacheConfig.keychainSharingGroup = keychainGroup;
```

Comment:
will this still use your bundleID internally?

Reply:
yes, `MSIDKeychainTokenCache` uses bundle id as keychain group to disable sharing if nil is passed :)